uppsrc/ide/Core/Util.cpp

```diff
@@ -25,3 +25,17 @@ int GetRepoKind(const String& p)
 	String pp = p;
 	return GetRepo(pp);
 }
+
+String GetGitPath()
+{
+#ifdef PLATFORM_WIN32
+	static String path;
+	ONCELOCK {
+		path = AppendFileName(GetExeFolder(), "\\bin\\mingit\\cmd\\git.exe");
+		path = FileExists(path) ? "\"" + path + "\"" : "git";
+	}
+	return path;

```

Comment:
I would firstly check if git executable is available in the PATH. After that, I would look for bundled git. Alternativly, we could use diffrent order. Firstly check for mingit and then if we don't find, it we should look for os wide git executable. 

The advantage of OS wide git executable is that it is always up to date with security patches on board. We can not tell the same for git executable bundled to mingit. We do not upgrade mingit that frequent.